sdk/schemaregistry/schema-registry-json/vitest.browser.config.ts

```diff
@@ -3,20 +3,14 @@
 
 import { defineConfig, mergeConfig } from "vitest/config";
 import viteConfig from "../../../vitest.browser.shared.config.ts";
-import browserMap from "@azure-tools/vite-plugin-browser-test-map";
-import inject from "@rollup/plugin-inject";
 
 export default mergeConfig(
   viteConfig,
   defineConfig({
-    optimizeDeps: {
```

Comment:
I'm curious, why do we remove browser injection in all the browser config files?

Reply:
My mistake, fixed.